stable/kommander/templates/grafana/opsportal-username-secret.yaml

```diff
@@ -36,7 +36,7 @@ metadata:
   annotations:
     "helm.sh/hook": pre-delete
     "helm.sh/hook-weight": "5"
-    "helm.sh/hook-delete-policy": hook-succeeded
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
```

Comment:
consistency nit

```suggestion
    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
```

Reply:
fixed and changed the ordering of the other hook in this file too